img/o/onet.pl.svg

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please use this optimised SVG instead:
![](https://raw.githubusercontent.com/ApCoder123/ApCoder123/master/5980.svg)